config/deploy/production.rb

```diff
@@ -0,0 +1,8 @@
+server 'sul-robots1-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
+server 'sul-robots2-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
```

Comment:
Just checking -- we're only interested in deployed to these two servers, and not any of the others?